src/main/java/com/aws/greengrass/util/platforms/Exec.java

```diff
@@ -140,24 +107,24 @@ public Exec logger(Logger logger) {
         return this;
     }
 
-    public static String cmd(String... command) throws InterruptedException, IOException {
-        return new Exec().withExec(command).execAndGetStringOutput();
+    public String cmd(String... command) throws InterruptedException, IOException {
```

Comment:
can we clean up some of these overrides? They get so complicated; way too many ways to call a command. I'd basically only keep what is used by ShellRunner + one method to run without a shell.

Reply:
Agreed there's too many ways. Added TODO to reduce this PR scope a bit